lib/elements/prompt.js

```diff
@@ -39,8 +39,8 @@ class Prompt extends EventEmitter {
       if (this.in.isTTY) {
         this.in.setRawMode(false);
       } else {
-        // we can't use `utils.clear()`, because it expect `prompt` string
-        this.out.write(erase.line + cursor.prevLine());
+        this.clear = clear(this.prompt)
```

Comment:
`this.prompt` should be available, because `this.close` is called after the `this.render` inside the `submit()`...